src/dev/build/tasks/clean_tasks.js

```diff
@@ -283,11 +279,26 @@ export const CleanCtagBuildTask = {
         }
 
         const ctagsBuildDir = build.resolvePathForPlatform(platform, RELATIVE_CTAGS_BUILD_DIR);
-        await deleteAll([
-          resolve(ctagsBuildDir, '*'),
-          `!${resolve(ctagsBuildDir, `ctags-node-v${process.versions.modules}-${getPlatformId(platform)}-x64`)}`
-        ], log);
+        await deleteAll(
+          [
+            resolve(ctagsBuildDir, '*'),
+            `!${resolve(
+              ctagsBuildDir,
+              `ctags-node-v${process.versions.modules}-${getPlatformId(platform)}-x64`
+            )}`,
+          ],
+          log
+        );
       })
     );
-  }
+  },
+};
+
+export const CleanIrrelevantFilesFromBuildTask = {
```

Comment:
Works for me - in the future we will probably combine this with the CleanNodeModules but I will take care of that.